### PR TITLE
Add a task test-coverage to generate a coverage report

### PR DESCRIPTION
**Pull request by boxydog**
# Pull Request Checklist

- [X] Ensured **tests pass** and (if applicable) updated functional test output
- [X] Conformed to **code style guidelines** by running appropriate linting tools
- [X] Added **tests** for changed code
- [X] Updated **documentation** for changed code

### CODE REVIEW

File: docs/contribute.rst [hunk 1]
```diff
@@ -75,6 +75,9 @@ via::
 
     invoke tests
 
+(For more on invoke, see `invoke -l` to list tasks, or
+[pinvoke.org](pyinvoke.org) for documentation.)
```
Comment (justinmayer) [inline docs/contribute.rst:79]: This is reST, not Markdown, so you'll need double-ticks. Also, I'd include the scheme. In short:
```
(For more on Invoke, see ``invoke -l`` to list tasks, or
[pyinvoke.org](https://pyinvoke.org) for documentation.)

File: tasks.py [hunk 2]
```diff
@@ -44,6 +44,13 @@ def tests(c):
     c.run(f"{VENV_BIN}/pytest", pty=PTY)
 
 
+@task
+def test_coverage(c):
```
Comment (justinmayer) [inline tasks.py:48]: Any reason not to name this task `coverage`? (Shorter, doesn't cause contention with `inv tests` completion)

**Review by justinmayer — CHANGES_REQUESTED** [review]
Thanks, @boxydog! I made a few corrections and a suggestion.

File: docs/contribute.rst (hunk 1 above)
Comment (justinmayer) [reply docs/contribute.rst:79]: Oops. Because it's reST, you'll also need to change the link syntax.

File: tasks.py (hunk 2 above)
Comment (boxydog) [reply tasks.py]: Sure, changed to "coverage"

File: docs/contribute.rst (hunk 1 above)
Comment (boxydog) [reply docs/contribute.rst]: Ah thanks. Changed to accept your comment, and look okay in rST (the `[pyinvoke.org]` didn't work).

File: tasks.py [hunk 3]
```diff
@@ -44,6 +44,13 @@ def tests(c):
     c.run(f"{VENV_BIN}/pytest", pty=PTY)
 
 
+@task
+def coverage(c):
+    """Generate code coverage of running the test suite"""
```
Comment (justinmayer) [inline tasks.py:49]: Sorry! One more thing… I think the linter will complain (not yet, but at some point in the future) without a period at the end of the doc-string, so perhaps get ahead of that and add it now?

Comment (boxydog) [reply tasks.py]: Added, just to get this merged.

But, I wouldn't bother with stuff like that. If the linter isn't complaining, leave it.

Then, fix the linter, and fix all the stuff at once. Every other task in this file doesn't have a period, so there will be a lot of stuff to fix.

**Review by justinmayer — APPROVED** [review]
Thank you for the contribution, @boxydog! ✨

File: tasks.py (hunk 3 above)
Comment (justinmayer) [reply tasks.py]: Fair point, and duly noted 😊

**Pull request merged**